File: nathan/Problema0417/problema0417.js
const prompt = require("prompt-sync")({ sigint: true });

function converterRad(x){
    return x * (3.14/180);
}

var graus = Number(prompt("Digite o valor em graus para a conversão em radianos: "));

var rad = converterRad(graus);
console.log("Radianos: " + rad);

File: nathan/Problema0417/problema0417.java
package Problema0417;

import java.util.Scanner;

public class problema0417 {

    static double converterRad(double graus){
        return graus * (3.14/180);
    }
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        
        System.out.println("Digite o valor em graus para a conversão em radianos: ");
        double graus = Double.valueOf(input.nextLine());

        double rad = converterRad(graus);
        System.out.println("Radianos: " + rad);
    }
}
